src/packages/sdk/src/utils/loadEnv.ts

```diff
@@ -16,97 +17,62 @@ type CLIArgs =
       '--telemetry-information': StringConstructor
     }>
 
-interface DotenvResult {
-  error?: Error
-  parsed?: {
-    [name: string]: string
-  }
-}
+type DotenvResult = dotenv.DotenvConfigOutput & {
+  ignoreProcessEnv?: boolean | undefined;
+};
 
 interface LoadEnvResult {
   message: string
-  dotenvResult: DotenvResult
+  path: string
+  dotenvResult:  DotenvResult
 }
-
+/**
+ * Tries load env variables
+ * 1. Checks schema dir
+ *    1. From --schema arg
+ *    2. PackageJSON schema config
+ *    3. Default Prisma Dir `./prisma`
```

Comment:
The numbers are confusing, should it be 1,2,3,4 ?

Reply:
Yeah that's old will change it